neurons/validators/scraper_validator.py

```diff
@@ -251,6 +253,86 @@ async def run_task_and_score(
 
         return async_responses, uids, event, start_time
 
+    async def run_task_and_score_with_uids(
```

Comment:
You can use same run_task_and_score function and add initial_uids as optional field

Reply:
I just updated code to use run_task_and_score with initial uids